apps/services/notifications/src/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const CONFIG_PROVIDER = 'CONFIG'
+export const CONNECTION_PROVIDER = 'CONNECTION'
```

Comment:
the time constants are not used outside of 1 single file and could cause confusion/errors since aws sqs could represent 1 second differently than other libraries (second  being 1 vs 1000)

As we don't need the constants anywhere else I think is safer to not expose them or make them re-usable

Reply:
See your point, marking this as resolved